### ISSUE TITLE
Middle-markers are not on the line

### ISSUE THREAD

**Issue by glebshalyganov** [mstahv/v-leaflet-editable#13]
Hello!
When i open EditingTest and zoom in to a triangle-polygon, i can see that middle-markers are shifting away from the line:
![image](https://user-images.githubusercontent.com/15035316/50217786-2a8dcf00-03a3-11e9-976f-c7e29f7edc2d.png)
I've noticed that if I first zoom to max value and then add editable feature to a map, than all works fine. It looks like it has a deal with a zooming.
I've found a similar issue in the Leaflet.Editable: https://github.com/Leaflet/Leaflet.Editable/issues/64.

**Comment by mstahv** [comment]
Can you easily reproduce that?

**Comment by glebshalyganov** [comment]
Yes, it reproduces every single time. Do you mean that it doesn't reproduce on your environment?
I guess it is a problem with the js lib. It seems to me that position of middle-markers (which is calculated in pixels) calculates only once when they displayed for the first time and then doesn't recalculate on zoom.

**Comment by glebshalyganov** [comment]
I've raised an issue on the leaflet addon: https://github.com/Leaflet/Leaflet.Editable/issues/182
